specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/bastionHost.json

```diff
@@ -404,6 +625,219 @@
         }
       },
       "description": "Response for ListBastionHosts API service call."
+    },
+    "BastionShareableLink": {
+      "properties": {
+        "vm": {
+          "$ref": "#/definitions/VM",
+          "description": "Reference of the virtual machine resource."
+        },
+        "bsl": {
```

Comment:
can we call this 'sharableLink' or just 'link'? we try to avoid acronyms and bastion is already part of the path.

Reply:
This question also doesn't appear to be addressed.

---
In reply to: [364407873](https://github.com/Azure/azure-rest-api-specs/pull/7969#discussion_r364407873) [](ancestors = 364407873)